backend/danswer/one_shot_answer/answer_question.py

```diff
@@ -229,6 +233,21 @@ def stream_answer_objects(
         else no_gen_ai_response()
     )
 
+    if qa_model is not None and query_req.return_contexts:
```

Comment:
Wasn't 100% sure if this is the best place for this if. On the one hand I didn't want to pass this `return_context` argument few more levels deeper to the llm call, but on the other hand right now I guess we can't really be sure if the `DanswerContexts` is correct - contexts that were actually passed to the LLM might be e.g. limited / transformed depending on the `qa_model.answer_question_stream` implementation

Reply:
Ya, looks good, this is fine :)